django_code/program/static/program/program.css

```diff
@@ -136,15 +133,21 @@ body {
 
 
 #sidebar {
-    padding: 0;
+    padding: 30px;
```

Comment:
it may be too much, but 0 doesn't sound good. I don't know what that looks like, so go ahead and change it. I will check it out and see if 0 is too little later.